lib/project_types/script/layers/infrastructure/languages/wasm_task_runner.rb

```diff
@@ -19,6 +21,25 @@ def dependencies_installed?
           def library_version(_library_name)
             nil
           end
+
+          def compiled_type
+            "wasm"
+          end
+
+          def metadata_file_location
+            "metadata.json"
+          end
+
+          def build
+            bytecode
+          end
+
+          private
+
+          def bytecode
+            raise Errors::WebAssemblyBinaryNotFoundError unless ctx.file_exist?(BYTECODE_FILE)
+            ctx.binread(BYTECODE_FILE)
+          end
```

Comment:
Nit: Since `build` only calls `bytecode` which has a couple lines, it might be a bit more readable to just put those two lines into `build` directly.

Reply:
Sounds fine to me!  I had split it up this way as I was mimicking how the TS and AS taskrunners are setup, but there's no reason to mimic the private implementation.